### PR TITLE
Aws s3 integration and sell item function

### PR DESCRIPTION
**Pull request by chrisstaudinger**
Aws s3 integration and sell item function
Tested and working

### CODE REVIEW

File: app/controllers/items_controller.rb [hunk 1]
```diff
@@ -16,6 +16,30 @@ def show
...
+    Measurement.all.each do |measurement|
+      @measurement_options << measurement.unit
+    end
+
+    @category_options = []
+    ItemCategory.all.each do |category|
+      @category_options << category.name
+    end
+
+    # def return_item_id_from_item_name(item_category_name)
```
Comment (sukibeww) [inline app/controllers/items_controller.rb]: remove this commented function, if it is no longer used.

File: app/controllers/items_controller.rb [hunk 2]
```diff
@@ -27,6 +51,8 @@ def edit
   def create
     @item = Item.new(item_params)
 
+    puts @item
```
Comment (sukibeww) [inline app/controllers/items_controller.rb]: is this diagnosis puts?

File: app/helpers/items_helper.rb [hunk 3]
```diff
@@ -1,2 +1,19 @@
 module ItemsHelper
+    # def return_item_id_from_item_name(item_category_name)
```
Comment (sukibeww) [inline app/helpers/items_helper.rb]: remove this commented function, if it is no longer used.

File: app/controllers/items_controller.rb (hunk 2 above)
Comment (chrisstaudinger) [reply app/controllers/items_controller.rb]: yep, I'll remove it now that it's working

File: app/controllers/items_controller.rb (hunk 1 above)
Comment (chrisstaudinger) [reply app/controllers/items_controller.rb]: yep

**Pull request merged**